docs(database): Be more explicit about operators/functions in CDS files (#632)
cap-js/docs: guides/databases-sqlite.md

* `year` `month`, `day`, `hour`, `minute`, `second` — return parts of a datetime

> <sup>1</sup> Argument `n` is optional.

> > These functions are only supported at runtime via the database service, but in CDS files.

Reviewer: Should it say "not"?
```suggestion
> These functions are only supported at runtime using the database service, but not in CDS files.
```

Author: Yes, thanks! 🤦‍♂️

> > These functions are only supported at runtime using the database service, but not in CDS files.

Reviewer 2: ```suggestion
> These functions are only supported within runtime queries, but not in CDS files.
```
